packages/linux-driver-addons/dvb/crazycat/patches/amlogic/driver.dvb.crazycat-04-AML-hacks.patch

```diff
@@ -11,6 +11,14 @@
 +	# Copy videobuf-res module
 +	cp -a "$(SRCDIR)/drivers/media/v4l2-core/videobuf-res.c" "drivers/media/v4l2-core/"
 +	cp -a "$(SRCDIR)/include/media/videobuf-res.h" "include/media/"
++	mkdir -p  drivers/media/dvb-avl/
++	# Copy dvb_tv driver
++	cp -a $(SRCDIR)/drivers/amlogic/dvb-avl "drivers/media/"
++	echo "obj-y += dvb-avl/" >> "drivers/media/Makefile"
++	# Copy wetekdvb driver
++	@if [ $(WETEKSRCDIR) != "" ]; then cp -a $(SRCDIR)/drivers/amlogic/wetek/ "drivers/media/"; fi
++	@if [ $(WETEKSRCDIR) != "" ]; then cp -a $(WETEKSRCDIR)/driver/wetekdvb_play2.ko "../v4l/wetekdvb.ko"; fi
```

Comment:
The other issue I see is that wetekdvb modules are prebuilt by Wetek and they will not work for crazycat or hauppauge.
I suggest that we update the wetekdvb package upstream to include a subdirectory "media_tree" with those modules precompiled for crazycat and hauppauge. @codesnake can you create a source package like that so @afl1 can add a commit to bump wetekdvb package and change the path's: to something like: "cp -a $(WETEKSRCDIR)/driver/media_tree/wetekdvb_play2.ko ...."